setup.cfg

```
@@ -48,7 +48,16 @@ install_requires =
     nilearn>=0.7.1
     numba  # used by sparse
     numpy
+<<<<<<< HEAD
+<<<<<<< HEAD
+    pandas
+    patsy  
+=======
     pandas>=1.1.0
+>>>>>>> 87c3ce30c59382605fd141c6149be25be742be96
+=======
+    pandas>=1.1.0
+>>>>>>> 87c3ce30c59382605fd141c6149be25be742be96
```

This is a bad merge.

```suggestion
    pandas>=1.1.0
    patsy  
```